src/client/ui/highlight.js

```diff
@@ -0,0 +1,26 @@
+export default function highlight(data) {
```

Comment:
couldn't find a module that did only this. Most of them were overkill as they had separate CSS and were made to parse different languages.